src/main/scala/org/clulab/wm/eidos/context/GeoPhraseID.scala

```diff
@@ -0,0 +1,5 @@
+package org.clulab.wm.eidos.context
+
+case class GeoPhraseID(phraseID: String, PhraseGeoID: Option[Int], StartOffset_locs: Int, EndOffset_locs: Int)
```

Comment:
@vikas95, the names of this class and its variables are a bit confusing.  Could you change them after reviewing and merging Keith's changes.

```suggestion
case class GeoLocation(text: String, geonameID: Option[Int], startOffset: Int, endOffset: Int)
```

Reply:
> @vikas95, the names of this class and its variables are a bit confusing. Could you change them after reviewing and merging Keith's changes.
> 
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
> case class GeoPhraseID(phraseID: String, PhraseGeoID: Option[Int], StartOffset_locs: Int, EndOffset_locs: Int)
> case class GeoLocation(text: String, geonameID: Option[Int], startOffset: Int, endOffset: Int)

Yes, I think I have to change few more variable names, I'll work on these.